apps/system/style/system/system.css

```diff
@@ -35,6 +35,8 @@ textarea,
   background-size: cover; /* Possible memory over-consumption with huge images, it should be removed for improving stability*/
   background-repeat: no-repeat;
   background-position: 50% 50%;
+
+  --devtools-scale: 0.4;
```

Comment:
nits: maybe express that in percentage, as all the calc you do with this var are for percentage. It would spare you one calc and simplify the other.